medocker/docker_shared/make_docker.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/sh
+# This must be run from the local directory containing the Dockerfile!!!
+# Requires an argument of the tag to provide, usually the git branch, and a
+# suffix, usually the image type.
+set -e
+
+if [ -z "$1" ]
+then
+    echo "No argument supplied for the docker tag (usually current git branch)"
+    exit 101
+fi
+
+DTAG=$1
+
+# Per https://serverfault.com/a/382740 this accepts the empty string, but not
+# no argument.
+if [ -z "${2+set}" ]
+then
+    echo "No argument supplied for the docker suffix (image type)"
+    exit 101
+fi
+
+SUFFIX=$2
+
+FULLTAG="$1:$2"
+
+if [ ! -d "../../../ME_warp" ]; then
+    echo "No ME_warp install found in ../../.."
+    exit 111
+elif [ ! -d "../../metools" ]; then 
+    echo "No metools install found in ../.."
+    exit 111
+elif [ ! -d "../../../minerva" ]; then 
+    echo "No minerva install found in ../../.."
+    exit 111
+fi
```

Comment:
Why is this needed? We shouldn't need any of those packages, no?

Reply:
Good catch. We just copied over all the docker stuff from warp and modified them, and we didn't use this file, and I left it in by accident.